### PR TITLE
adds the dataset description for the machine learning dataset for sdo

### PR DESCRIPTION
**Pull request by mariusgiger**

### CODE REVIEW

File: datasets/machine-learning-dataset-for-sdo/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+The data is stored as follows:
+
+- AIA: as a set of images of the form ({$year}/AIA/{$wavelength}/{$month}/{$day}/AIA{$year}{$month}{$day}_{$hour}{$minute}_{$wavelength}.npz) e.g., AIA/2013/AIA/0211/11/25/AIA20131125_0824_0211.npz
+- HMI : as a set of images of the form ({$year}/HMI/{$wavelength}/{$month}/{$day}/HMI{$year}{$month}{$day}_{$hour}{$minute}_{$wavelength}.npz) e.g., HMI/2013/HMI/bz/11/25/HMI20131125_0824_bz.npz
+- EVE: as a single numpy file EVE/irradiance.npy, where each row is a date in time. Any invalid datapoint is set as -1, which you should specially handle or delete. The ones primarily of interest are from MEGS-A: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 14] (note the missing 13); Indices 15 and higher are rarely observed.
+
+Each image is stored as a variable 'x' in each npz. For compression reasons, the data is stored as float16 if the data fits and float32 otherwise. You should immediately, however, convert the data to at least float32.
+
+The instruments' data are joined by join.csv, which has the form
```
Comment (mariusgiger) [inline datasets/machine-learning-dataset-for-sdo/README.md]: @yashtyk did you find join.csv somewhere? I cannot find it but they mention it in their downsized version of the dataset: https://github.com/dfouhey/sdodemo

Comment (yashtyk) [reply datasets/machine-learning-dataset-for-sdo/README.md]: @mariusgiger for the downsized version there is one here http://fouheylab.eecs.umich.edu/~fouhey/MiniSDO/ in EVEPlusMeta.tar

**Review by yashtyk — APPROVED** [review]

File: datasets/machine-learning-dataset-for-sdo/README.md (hunk 1 above)
Comment (yashtyk) [reply datasets/machine-learning-dataset-for-sdo/README.md]: cannot find the same for the whole dataset

Comment (mariusgiger) [reply datasets/machine-learning-dataset-for-sdo/README.md]: ok, yeah I did not find it either - I will remove this section

**Pull request merged**